docs/_includes/calendar.html

```diff
@@ -0,0 +1,396 @@
+    {% endhighlight %}
+
+    <h3>Interactive Calendar</h3>
+
+    <p>To make a calendar interactive, replace the <code>span</code> elements with <code>button</code>. Buttons are disabled directly instead of with a class, but all other attributes remain the same between static and interactive calendars. In interactive calendars, one and only one button should have a <code>tabindex</code> in the table. The rest need to set <code>tabindex</code> to <code>-1</code>.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="calendar">
+                <div class="calendar__body">
+                    <div class="calendar__month">
+                        {% include calendar-month.html interactive="true" index="1" %}
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    {% highlight html %}
+<div class="calendar">
+    <div class="calendar__body">
+        <div class="calendar__month">
+            <table>
+                <caption>
+                    August 2019
+                </caption>
+                <thead>
+                    <tr>
+                        <th>Su</th><th>Mo</th><th>Tu</th><th>We</th><th>Th</th><th>Fr</th><th>Sa</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td colspan="4"></td>
+                        <td><button tabindex="-1" disabled>1</button></td>
+                        <td><button tabindex="-1" disabled>2</button></td>
+                        <td><button tabindex="-1" disabled>3</button></td>
+                    </tr>
+                    ...
+                    <tr>
+                        <td><button tabindex="-1" disabled>11</button></td>
+                        <td><button tabindex="-1" disabled>12</button></td>
+                        <td><button tabindex="-1" disabled>13</button></td>
+                        <td aria-current="date">
+                            <button>14</button>
+                        </td>
+                        <td><button tabindex="-1">15</button></td>
+                        <td><button tabindex="-1">16</button></td>
+                        <td><button tabindex="-1">17</button></td>
+                    </tr>
+                    ...
+                    <tr>
+                        <td><button tabindex="-1">25</button></td>
+                        <td><button tabindex="-1">26</button></td>
+                        <td><button tabindex="-1">27</button></td>
+                        <td><button tabindex="-1">28</button></td>
+                        <td><button tabindex="-1">29</button></td>
+                        <td><button tabindex="-1">30</button></td>
+                        <td><button tabindex="-1">31</button></td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    </div>
+</div>
+    {% endhighlight %}
+
+    <h3>Calendar with selected range</h3>
```

Comment:
We might want to bring the selected range example forward before interactive example (as a static calendar could also theoretically be used to display a range, I think?)

Reply:
I mention in the description that "Static and Dynamic Calendars" can both have range displays. Is that not clear enough?

Reply:
It's clear, just seems odd to have it here (after interactive calendar section) when the example shows a static calendar.

Reply:
That's fair, I chose static calendar because the markup is smaller but it is a little inconsistent. I'll move it up.